static/js/modules/analytics.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+/* global ga */
+
+var URI = require('URIjs');
+var _ = require('underscore');
+
+function init() {
+  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
+
+  ga('set', 'forceSSL', true);
+  ga('set', 'anonymizeIp', true);
+  ga('create', 'UA-48605964-22', 'auto');
+}
+
+function pageView() {
+  if (typeof ga === 'undefined') { return; }
+  var path = document.location.pathname;
+  if (document.location.search) {
+    var query = URI.parseQuery(document.location.search);
+    path += '?' + sortQuery(query);
+  }
+  ga('send', 'pageview', path);
+}
+
+function sortQuery(query) {
```

Comment:
Not sure how to test this. @msecret would you mind just peeking at this function and let us know if it makes sense?